### PR TITLE
updated the new design for question 4

### PR DESCRIPTION
**Pull request by Amutha37**
This is the latest update with the new design for question 4. I have made some changes on <LinkButton > to get the question for working right.

### CODE REVIEW

File: pages/form/Question2.jsx [hunk 1]
```diff
@@ -2,9 +2,9 @@ import { useState } from "react";
 import {
   Box,
   Grid,
-  Heading,
+  Heading, 
   Text,
-  Select,
+  Select,
```
Comment (hqtan) [inline pages/form/Question2.jsx:7]: please remove trailing space

```suggestion
  Select, 
```

File: pages/form/Question4.jsx [hunk 2]
```diff
@@ -1,34 +1,92 @@
...
-      />
-      <Grid templateColumns="repeat(2, 1fr)" gap={4}>
-        <BackButton href="/form/Question3" onClick={saveAnswers} />
-        <ContinueButton href="/form/Question5" onClick={saveAnswers} />
-      </Grid>
+      {/* Responsive buttons */}
+      <Flex
+        maxWidth="full"
+        margin={"60px 0px 50px"}
+        // flexDirection={["column", "row"]}
```
[the remark this answers is not in the document]
Comment (Amutha37) [reply pages/form/Question4.jsx]: I have reinstate that line line

File: pages/form/Question4.jsx [hunk 3]
```diff
@@ -1,34 +1,100 @@
...
+      >
+        Select the main way you travel to work
+      </Heading>
+
+      {/* Responsive buttons */}
+      <Flex maxWidth="full" margin={"60px 0px 50px"} flexDirection="column">
+        <Box fontSize="20px" mr="10px" textAlign="left">
+          <Text my={2} maxWidth={"550px"}>
+            For example, if you drive 2km to the train and then catch the train
+            for 15km, choose 'Train' as your way of travel.
```
Comment (hqtan) [inline pages/form/Question4.jsx:59]: @Amutha37 , you need to be mindful of escaping the apostrophes in your text!

```suggestion
            for 15km, choose &apos;Train&apos; as your way of travel.
```

**Review by Amutha37 — COMMENTED** [review]
This icon now in use thank you.

File: pages/form/Question4.jsx [hunk 4]
```diff
@@ -1,34 +1,117 @@
...
+      </Box>
+
+      <Heading
+      // fontSize="40px"
+      // mt="6"
+      // fontWeight={700}
+      // maxWidth={"624px"}
+      // textAlign={"center"}
+      // marginBottom={"30px"}
+      >
```
Comment (batteries76) [inline pages/form/Question4.jsx:59]: Great, but looks like you can remove these now.

**Pull request merged**